src/utils.php

```diff
@@ -35,3 +35,37 @@ function create_list_of_cars(array $cars): string | false
 
     return $ret;
 }
+
+function validateDataFields($name, $year_made, $zero_sixty, $price, $brand): string | true
+{
+    $err_msg = "";
+    if (is_string($name) == false || empty($name) == true || strlen($name) > 30 || strlen($name) <= 0) {
```

Comment:
You can simplify these conditions slighty by removing unnecessary comparison operators. For example, this line can become:

```
if (!is_string($name) || empty($name) || strlen($name) > 30 || strlen($name) <= 0) {
```